addons/github/static/githubFangornConfig.js

```diff
@@ -192,6 +192,14 @@ var _githubItemButtons = {
                         className: 'text-info'
                     }, branchArray)
                 );
+            } else {
```

Comment:
@Johnetordoff So if I'm reading this correctly, you can't download the entire repo as a zip. Is that a) what this else block means; and b) correct?

Reply:
So the user can download the entire repo as a zip. They have always been able to to this via the normal download button on the root level directory. The "Download" button there actually functions as a download as zip button.

Here: https://github.com/CenterForOpenScience/osf.io/blob/develop/addons/github/static/githubFangornConfig.js#L236

To download the entire repo you presumably need to the branch as a parameter which is why this is a little strange. Didn't really want to change existing behavior, but upon reflection we should probably change the root level download button to "Download as Zip" to bring it in line.